docs/constants/index.rst

```diff
@@ -173,9 +173,12 @@ If either `astropy.constants` or `astropy.units` have already been imported, a
 To temporarily set constants to an older version (e.g.,
 for regression testing), a context manager is available, as follows:
 
+    >>> import warnings
```

Comment:
Here, let's remove the example altogether - we don't want people to find it!

Reply:
Good point!